compliance_checker/tests/test_cf.py

```diff
@@ -94,8 +93,13 @@ def test_check_data_types(self):
 
         dataset = self.load_dataset(STATIC_FILES['bad_data_type'])
         result = self.cf.check_data_types(dataset)
-        assert result.msgs[0] == u'The variable temp failed because the datatype is int64'
-        assert result.value == (6, 7)
+
+        # TODO
+        # the acdd_reformat_rebase branch has a new .nc file that I made
```

Comment:
Try to avoid use of the first person in technical writing.

Reply:
Good point -- this will be addressed for the next commit